src/Main.purs

```diff
@@ -313,7 +317,7 @@ Available options:
 -- Due to `catchException` label annoyingness
 catchException'
   :: forall a eff
-   . (Error -> Eff (err :: EXCEPTION | eff) a)
-  -> Eff (err :: EXCEPTION | eff) a
-  -> Eff (err :: EXCEPTION | eff) a
+   . (Error -> Eff (exception :: EXCEPTION | eff) a)
+  -> Eff (exception :: EXCEPTION | eff) a
+  -> Eff (exception :: EXCEPTION | eff) a
```

Comment:
This function can be removed altogether since we no longer have duplicate label issues.